src/python/system/environment.py

```diff
@@ -1056,6 +1056,11 @@ def is_android(plt=None):
   return 'ANDROID' in (plt or platform())
 
 
+def is_android_gce(plt=None):
```

Comment:
nit: rename to is_android_cuttlefish to be clearer.

Reply:
Done